portfolio/src/main/webapp/script.js

```diff
@@ -175,7 +175,8 @@ function fetchComments(createNewPagination = false) {
 }
 
 /**
- * Creates an <li> element containing a comment, including the username, content, time, and delete button.
+ * Creates an <li> element containing an individual comment. The element includes the 
+       following attributes associated with the comment: username, content, time, and delete button.
```

Comment:
the "following" should be vertically below the "Creates"

Reply:
ok